docs/snippets/unhook--remove-woocommerce-emails.md

```diff
@@ -0,0 +1,47 @@
+# Unhook/Remove WooCommerce Emails
```

Comment:
```suggestion
# Unhook and remove WooCommerce emails
```

Reply:
I changed the docs title according to https://github.com/woocommerce/woodocs/blob/trunk/style-guide.md:
> Use sentence case (not title case) for docs titles and subheadings. Example: “Introduction to the launch experience” rather than “Introduction to the Launch Experience.”